Jenkinsfile

```diff
@@ -4,7 +4,7 @@ node('linux && docker') {
   def isBump = !!tag
   def isMaster = env.BRANCH_NAME == 'master'
 
-  withEnv(['npm_config_cache=npm-cache', 'CI=true']) {
+  withEnv(['npm_config_cache=npm-cache', 'CI=true', 'NODE_OPTIONS=--max_old_space_size=4096']) {
```

Comment:
FYI @olamothe @ThibodeauJF

The build was failing, I checked with Sami and decided to try bumping the memory limit (it was 2gb, it now is 4gb) and the build is now succeeding

Just wanted to let you know :sweat_smile: